sshkey/sshkey_test.go

```diff
@@ -0,0 +1,48 @@
+package sshkey
+
+import (
+	"os"
+	"path/filepath"
+	"testing"
+
+	"github.com/bitrise-steplib/steps-activate-ssh-key/command"
+	"github.com/bitrise-steplib/steps-activate-ssh-key/log"
+	"github.com/stretchr/testify/assert"
+	"github.com/stretchr/testify/mock"
+)
+
+func TestAgentAddKey_CallsSSHAgent(t *testing.T) {
+	// Given Simple SSH Agent
```

Comment:
I think writing `// Given`should be enough in itself, as these are merely denoting parts of the test.